fork update erro

### 问题描述

failed to load config from /home/runner/work/blog2026/blog2026/vite.config.ts
这个error发生在Deply to Cloudflare Workers步骤中
代码如下：

### 复现步骤

1. failed to load config from /home/runner/work/blog2026/blog2026/vite.config.ts
error during build:
ZodError: [
  {
    "code": "invalid_value",
    "values": [
      "default",
      "fuwari"
    ],
    "path": [
      "THEME"
    ],
    "message": "Invalid option: expected one of \"default\"|\"fuwari\""
  }
]
    at file:///home/runner/work/blog2026/blog2026/node_modules/.vite-temp/vite.config.ts.timestamp-1771742268147-657cb17f1aeb6.mjs:147:35
    at loadConfigFromFile (file:///home/runner/work/blog2026/blog2026/node_modules/vite/dist/node/chunks/config.js:35766:64)
    at async resolveConfig (file:///home/runner/work/blog2026/blog2026/node_modules/vite/dist/node/chunks/config.js:35413:22)
    at async createBuilder (file:///home/runner/work/blog2026/blog2026/node_modules/vite/dist/node/chunks/config.js:33875:19)
    at async CAC.<anonymous> (file:///home/runner/work/blog2026/blog2026/node_modules/vite/dist/node/cli.js:629:10)
error: script "build" exited with code 1
Error: Process completed with exit code 1.
2. 
3.


### 预期结果

_No response_

### 环境信息/附件

_No response_

--- a/vite.config.ts
+++ b/vite.config.ts
@@ -12,13 +12,7 @@ import packageJson from "./package.json";
 import { themeNames, themes } from "./src/features/theme/config";
 
 const buildEnvSchema = z.object({
-  THEME: z.preprocess(
-    (val) => {
-      if (typeof val === "string" && val.trim() === "") return undefined;
-      return val;
-    },
-    z.enum(themeNames).default("default"),
-  ),
+  THEME: z.enum(themeNames).catch("default"),
 });
 
 const config = defineConfig(({ mode }) => {

